src/app/app.component.ts

```diff
@@ -20,7 +21,8 @@ export class AppComponent implements OnInit {
       error: (error) => {
         console.log('heroService.error', error);
       },
-      next: () => {
+      next: (hero) => {
```

Comment:
Change to `next: (hero: IHero) => {`

Reply:
why this change if the server can send any other type?

Reply:
Pipes are called before responses